recipes/protobuf/all/conanfile.py

```diff
@@ -229,9 +229,10 @@ def package_info(self):
                 self.cpp_info.components["libprotobuf"].defines = ["PROTOBUF_USE_DLLS"]
 
         # libprotoc
-        self.cpp_info.components["libprotoc"].set_property("cmake_target_name", "protobuf::libprotoc")
-        self.cpp_info.components["libprotoc"].libs = [lib_prefix + "protoc" + lib_suffix]
-        self.cpp_info.components["libprotoc"].requires = ["libprotobuf"]
+        if self.settings.os != "tvOS":
```

Comment:
`protoc` uses `libprotoc`, and `fork()` and `execv()` are in subprocess.cc, part of libprotoc.

Reply:
Correct, I've tested and libprotoc can't be build for tvOS